### PR TITLE
IDEMPIERE-5932 Update English top-level menu names and window names to better support new user adoption

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-5932

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [x] New and existing unit tests pass locally with my changes
- [x] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+-- Dec 8, 2023, 6:13:01 AM CST
+UPDATE AD_Menu SET Name='Payment and Reciept', Description='Process Payments and Receipts', Updated=TO_TIMESTAMP('2023-12-08 06:13:01','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=235
+;
+
+-- Dec 8, 2023, 6:15:33 AM CST
+UPDATE AD_Window SET Name='Sales Invoice and Note',Updated=TO_TIMESTAMP('2023-12-08 06:15:33','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=167
+;
+
+-- Dec 8, 2023, 6:15:33 AM CST
+UPDATE AD_Menu SET Name='Sales Invoice and Note', Description='Customer Invoice Entry', Updated=TO_TIMESTAMP('2023-12-08 06:15:33','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=178
```
Comment (hengsin) [inline migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:96]: Description is not in sync. Also, perhaps clearer to use **Sales Invoice and Credit/Debit Note** instead ?

File: migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+;
+
+-- Dec 8, 2023, 6:15:56 AM CST
+UPDATE AD_Window SET Name='Purchase Invoice and Note',Updated=TO_TIMESTAMP('2023-12-08 06:15:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=183
+;
+
+-- Dec 8, 2023, 6:15:56 AM CST
+UPDATE AD_Menu SET Name='Purchase Invoice and Note', Description='Vendor Invoice Entry', Updated=TO_TIMESTAMP('2023-12-08 06:15:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=206
```
Comment (hengsin) [inline migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:104]: Description is not in sync. Also, perhaps clearer to use Purchase Invoice and Credit/Debit Note instead ?

File: migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+-- Dec 8, 2023, 6:11:57 AM CST
+UPDATE AD_Menu SET Name='Window, Tab and Field', Description='Maintain Windows, Tabs & Fields', Updated=TO_TIMESTAMP('2023-12-08 06:11:57','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=141
+;
+
+-- Dec 8, 2023, 6:12:46 AM CST
+UPDATE AD_Window SET Name='Inventory Decrease/Increase',Updated=TO_TIMESTAMP('2023-12-08 06:12:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=341
+;
+
+-- Dec 8, 2023, 6:12:46 AM CST
+UPDATE AD_Menu SET Name='Inventory Decrease/Increase', Description='Enter Internal Use of Inventory', Updated=TO_TIMESTAMP('2023-12-08 06:12:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=503
```
Comment (hengsin) [inline migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:80]: Description is not in sync.

File: migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+-- Dec 8, 2023, 6:07:22 AM CST
+UPDATE AD_Menu SET Name='Requisition-to-Invoice (Purchasing)',Updated=TO_TIMESTAMP('2023-12-08 06:07:22','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=203
+;
+
+-- Dec 8, 2023, 6:08:15 AM CST
+UPDATE AD_Menu SET Name='Open Items (Financial movements and Aging)',Updated=TO_TIMESTAMP('2023-12-08 06:08:15','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=236
+;
+
+-- Dec 8, 2023, 6:08:21 AM CST
+UPDATE AD_Menu SET Name='Open Items (Aging)',Updated=TO_TIMESTAMP('2023-12-08 06:08:21','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=243
```
Comment (hengsin) [inline migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:20]: The report title is **Open Item (Invoice) List**, perhaps the two should be in sync ?

**Comment by CarlosRuiz-globalqss** [comment]
Ready @hengsin - I implemented your suggestions.

Testing this I also noticed that favourites were showing the name as tooltip, while the menu is showing the description as tooltip, so I fixed the favourite to show description too.

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**